scripts/integration-tests/e2e-jest.sh

```diff
@@ -47,6 +47,7 @@ if [ "$BABEL_8_BREAKING" = true ] ; then
 
   # Jest depends on @types/babel__traverse for Babel 7, and they contain the removed Noop node
   sed -i 's/t.Noop/any/g' node_modules/@types/babel__traverse/index.d.ts
+  sed -i 's/t.Noop/any/g' node_modules/@types/babel__traverse/ts4.1/index.d.ts
```

Comment:
`@types/babel__traverse` added type definitions for ts4.1+ separated from the "legacy ones", so we need to repeat the same hack to prevent them from relying on the removed `t.Noop`.